searchapp/static/student_view_style.css

```diff
@@ -218,4 +263,9 @@ input[type="file"] {
 	width:50%;
 	display:block;
 	margin-bottom:2%;
+}
+
+h3
+{
+	text-align:center;
 }
```

Comment:
Add a new line here. End all files with a new line. (I probably forgot to add here)